### DISCUSSION TITLE
Scroll in list not working on Samsung S22 Ultra Android 14

### DISCUSSION THREAD

**Discussion by antoshkow** [sanity-io/sanity#6596] [Sanity Studio Support]
Scrolling does not work on Samsung devices in the documentList
But It works on Apple devices
I updated the sanity-cli, as recommended in one of the requests, but it did not help :(
```
 "dependencies": {
"sanity": "^3.27.0",
```
```
 "devDependencies": {
    "@sanity/cli": "^3.27.1",
```

Thank you in advance for your help!

**Comment by Karytonn** [comment]
Any solution?
